backend/src/test/java/gov/cdc/usds/simplereport/service/ApiUserServiceTest.java

```diff
@@ -208,6 +214,30 @@ void createUserInCurrentOrg_disabledUserWrongOrg_error() {
     assertEquals("Unable to add user.", caught.getMessage());
   }
 
+  @Test
+  @WithSimpleReportSiteAdminUser
+  void getAllActiveUsersByOrganization_success() {
+    Organization org = _dataFactory.createValidOrg();
+    _dataFactory.createValidApiUser("allfacilities@example.com", org);
+    _dataFactory.createValidApiUser("nofacilities@example.com", org);
+    UserInfo userToBeDeleted = _dataFactory.createValidApiUser("somefacilities@example.com", org);
+    // delete a user
+    _service.setIsDeleted(userToBeDeleted.getInternalId(), true);
+    // create another org with a user
+    Organization differentOrg = _dataFactory.createValidOrg("other org", "k12", "OTHER_ORG", true);
```

Comment:
Are you creating this just to show that its users don't get returned in the call with `org` below?

Reply:
I think it's a good test, maybe just explicitly assert on it not being there to make the connection clear. This isn't absolutely necessary, I just prefer it to keep the tests readable so when someone reads through they don't have to think twice about why something is included.